### ISSUE TITLE
Does not save game difficulty

### ISSUE THREAD

**Issue by patrakov** [Frunit/qpop.js#5]
To reproduce:

1. Play at the "everyone is Charles Darwin" difficulty level.
2. Win or lose, does not matter.
3. Get greeted with the difficulty selection screen again, with everything at the boring "Darwin's Aunt" level.

Expected result: everything at the difficulty levels used in the previous game.

**Comment by Frunit** [comment]
Good idea! I don't have the time now, but I will hopefully fix this soon.
